pkg/config/config_template.yaml

```diff
@@ -1612,6 +1612,35 @@ api_key:
   #   - HISTSIZE
   #   - HISTFILESIZE
 
+  ## @param activity_dump - custom object - optional
+  ## Activity dumps
+  #
+  # activity_dump:
+
+    ## @param enabled - boolean - optional - default: false
+    ## @env DD_RUNTIME_SECURITY_CONFIG_ACTIVITY_DUMP_ENABLED - boolean - optional - default: false
+    ## Set to true to activate the security profiles feature.
+    #
+    #  enabled: false
+
+    ## @param traced_cgroups_count - integer - optional - default: 5
+    ## @env DD_RUNTIME_SECURITY_CONFIG_ACTIVITY_DUMP_TRACED_CGROUPS_COUNT - integer - optional - default: 5
+    ## Defines the number of concurrent cgroups to be learned.
+    #
+    #  traced_cgroups_count: 5
+
+    ## @param cgroup_dump_timeout - integer - optional - default: 30
+    ## @env DD_RUNTIME_SECURITY_CONFIG_ACTIVITY_DUMP_CGROUP_DUMP_TIMEOUT - integer - optional - default: 30
+    ## Defines the duration of cgroups learning phase in minutes. Min value is 10.
+    #
+    #  cgroup_dump_timeout: 30
+
+    ## @param cgroup_wait_list_timeout - integer - optional - default: 75
+    ## @env DD_RUNTIME_SECURITY_CONFIG_ACTIVITY_DUMP_CGROUP_WAIT_LIST_TIMEOUT - integer - optional - default: 75
+    ## Defines the timeout to wait before being able to learn again the same cgroup, in minutes.
+    #
+    #  cgroup_wait_list_timeout: 75
+
 {{ end -}}
 {{ end -}}
```

Comment:
Do you think we could add the network config in the same PR (like `runtime_security_config.network.enabled`) ?

Reply:
Let's do it!